lib/model/actor.js

```diff
@@ -0,0 +1,18 @@
+const { Subclass } = require('../util/util');
+const AsActor = require('./trait/actor');
+const AsActee = require('./trait/actee');
+const Base = require('./base');
+
+
+const actorTypes = { system: 'system', user: 'user', group: 'group', proxy: 'proxy' };
```

Comment:
I think I understand the other types, but what's a proxy?

Reply:
It's for limited-access revokable tokens on a subset of a user's permissions. To try at some security in an HTTP context.

Reply:
Yes. Optional/v2 type stuff.